Libraries/config/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright (c) 2015-present, Facebook, Inc.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree. An additional grant
+# of patent rights can be found in the PATENTS file in the same directory.
+#
+
+add_library(config SHARED
+            Sources/Config.cpp
```

Comment:
Yeah, that's true. I guess my thought is that since it references platforms and toolchains in the configuration, that it's already sort of related.

I'm also not sure if configuration makes sense in general since it wouldn't be compatible with anything from Xcode generally. My thought was that coupling it closer might reduce the chances of using it when it's not appropriate if it requires more conscious effort.